### PR TITLE
Fix plugin configuration path

### PR DESCRIPTION
**Pull request by smortex**
The dashboard has references to an old path that does not exist anymore:
`plugins/opensearch-security/securityconfig/config.yml`.  It was
replaced by `config/opensearch-security/config.yml` in this commit:

https://github.com/opensearch-project/security/commit/b44d7eafc1250a0d58828ef6f9cef48b5755fefc

Fix the last occurences of `plugins/opensearch-security/securityconfig`.

### CODE REVIEW

**Review by cwperks — DISMISSED** [review]

**Comment by cwperks** [comment]
Thank you for the contribution @smortex! The change looks good to me.

File: public/apps/configuration/panels/auth-view/instruction-view.tsx [hunk 1]
```diff
@@ -35,7 +35,7 @@ export function InstructionView(props: { config: ClientConfigType }) {
       <EuiText textAlign="center" size="xs" color="subdued" className="instruction-text">
         In order to use Security plugin, you must decide on authentication <EuiCode>authc</EuiCode>{' '}
         and authorization backends <EuiCode>authz</EuiCode>. Use{' '}
-        <EuiCode>plugins/opensearch-security/securityconfig/config.yml</EuiCode> to define how to
+        <EuiCode>config/opensearch-security/config.yml</EuiCode> to define how to
```
Comment (peternied) [inline public/apps/configuration/panels/auth-view/instruction-view.tsx:38]: Looks like there is a code formatting issue on this line

**Review by peternied — CHANGES_REQUESTED** [review]
Thanks for the contribution, we are really close to merging this pull request.   It looks like the continuous integrations checks are failing for this change, please take a look.

File: public/apps/configuration/panels/auth-view/instruction-view.tsx (hunk 1 above)
Comment (smortex) [reply public/apps/configuration/panels/auth-view/instruction-view.tsx]: Should be fixed!

**Comment by smortex** [comment]
I amended the commit with the needed change and force pushed to update the PR.  CI still needs to be approved, thanks!

**Comment by RyanL1997** [comment]
> I amended the commit with the needed change and force pushed to update the PR. CI still needs to be approved, thanks!

Thanks for the recommit, and I just re-run the CI.

**Review by RyanL1997 — APPROVED** [review]

**Review by cwperks — APPROVED** [review]

**Review by DarshitChanpura — APPROVED** [review]

**Review by scrawfor99 — APPROVED** [review]
Looks good to me!

**Review by peternied — APPROVED** [review]

**Pull request merged**